dodge_the_creeps_3d_c#/.gitignore

```diff
@@ -0,0 +1,2 @@
+.import
+logs/
```

Comment:
This file is not necessary (and actually should be deleted from the GDScript version too).